File: module06/module212/src/main/java/com/android/example/module06_module212/ClassAAJ.java
package com.android.example.module06_module212;

public class ClassAAJ {
	private 	com.android.example.module06_module340_module3.ClassAAB instance_var_1_0 = new com.android.example.module06_module340_module3.ClassAAB();
	private 	com.android.example.module06_module340_module3.ClassAAA instance_var_1_1 = new com.android.example.module06_module340_module3.ClassAAA();
	private 	com.android.example.module06_module340_module3.ClassAAJ instance_var_1_2 = new com.android.example.module06_module340_module3.ClassAAJ();
	private 	com.android.example.module06_module340_module3.ClassAAB instance_var_1_3 = new com.android.example.module06_module340_module3.ClassAAB();
	private 	com.android.example.module06_module340_module3.ClassAAA instance_var_1_4 = new com.android.example.module06_module340_module3.ClassAAA();

	public void method0(
		com.android.example.module06_module340_module3.ClassAAB param0,
		com.android.example.module06_module340_module3.ClassAAG param1,
		com.android.example.module06_module340_module3.ClassAAI param2,
		com.android.example.module06_module340_module3.ClassAAI param3) throws Throwable {
		param0.method2("SomeString", "SomeString");

		androidx.core.os.CancellationSignal local_var_2_4 = new androidx.core.os.CancellationSignal();
		local_var_2_4.throwIfCanceled();

	}

	public void method1(
		com.android.example.module06_module340_module3.ClassAAE param0) throws Throwable {
		androidx.core.os.CancellationSignal local_var_2_1 = new androidx.core.os.CancellationSignal();
		local_var_2_1.cancel();

		androidx.core.os.CancellationSignal local_var_2_2 = new androidx.core.os.CancellationSignal();
		local_var_2_2.throwIfCanceled();

		androidx.lifecycle.MethodCallsLogger local_var_2_3 = new androidx.lifecycle.MethodCallsLogger();
		local_var_2_3.approveCall("SomeString", 67);

		androidx.core.os.CancellationSignal local_var_2_4 = new androidx.core.os.CancellationSignal();
		local_var_2_4.cancel();

	}

	public void method2(
		com.android.example.module06_module340_module3.ClassAAG param0,
		com.android.example.module06_module340_module3.ClassAAI param1,
		com.android.example.module06_module340_module3.ClassAAG param2) throws Throwable {
	}
}

File: module06/module002/module4/src/main/java/com/android/example/module06_module002_module4/ClassAAJ.kt
package com.android.example.module06_module002_module4

@Suppress("UNUSED_PARAMETER")
class ClassAAJ {
	private 	val instance_var_1_0: com.android.example.module06_module002_module5.ClassAAC = com.android.example.module06_module002_module5.ClassAAC()
	private 	val instance_var_1_1: com.android.example.module06_module002_module1.ClassAAD = com.android.example.module06_module002_module1.ClassAAD()
	private 	val instance_var_1_2: com.android.example.module06_module002_module5.ClassAAH = com.android.example.module06_module002_module5.ClassAAH()
	private 	val instance_var_1_3: com.android.example.module06_module002_module1.ClassAAC = com.android.example.module06_module002_module1.ClassAAC()
	private 	val instance_var_1_4: com.android.example.module06_module002_module1.ClassAAD = com.android.example.module06_module002_module1.ClassAAD()

	fun method0(
		param0: com.android.example.module06_module002_module5.ClassAAE
	) {
		param0.method0(dagger.internal.DelegateFactory(), dagger.internal.DelegateFactory(), dagger.internal.DelegateFactory())

		for (iAb in 0..3) {
			val local_var_3_0: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
			local_var_3_0.hasSubscription()

			val local_var_3_1: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
			local_var_3_1.cancel()

			val local_var_3_2: androidx.core.os.CancellationSignal = androidx.core.os.CancellationSignal()
			local_var_3_2.cancel()

			local_var_3_0.isCancelled()

		}
	}

	fun method1(
		param0: com.android.example.module06_module002_module5.ClassAAH,
		param1: com.android.example.module06_module002_module1.ClassAAH,
		param2: com.android.example.module06_module002_module1.ClassAAH,
		param3: com.android.example.module06_module002_module5.ClassAAB
	) {
		val local_var_2_4: androidx.lifecycle.MethodCallsLogger = androidx.lifecycle.MethodCallsLogger()
		local_var_2_4.approveCall("SomeString", 32)

		param0.method1(io.reactivex.internal.disposables.ListCompositeDisposable(), io.reactivex.subscribers.TestSubscriber(), io.reactivex.subscribers.TestSubscriber(), dagger.internal.DelegateFactory())

		val local_var_2_5: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_5.cancel()

		val local_var_2_6: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_6.isCancelled()

	}

	fun method2(
		param0: com.android.example.module06_module002_module5.ClassAAB,
		param1: com.android.example.module06_module002_module5.ClassAAF
	) {
		val local_var_2_2: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_2.onComplete()

	}

	fun method3(
		param0: com.android.example.module06_module002_module5.ClassAAC,
		param1: com.android.example.module06_module002_module5.ClassAAG,
		param2: com.android.example.module06_module002_module5.ClassAAC
	) {
	}
}
